test/e2e/edu-5490-helpers.js

```diff
@@ -0,0 +1,150 @@
+import _ from 'lodash/fp';
+const Timeouts = require('./timeouts.js');
+
+
+// Same as 1990e
+// TODO: relocate to keep it DRY
+function completeEmploymentHistory(client, data, onlyRequiredFields) {
+  if (!onlyRequiredFields) {
+    const nonMilitaryJobs = data.nonMilitaryJobs[0];
+    client
+      .click('input[name="root_view:hasNonMilitaryJobsYes"]')
+      .clearValue('input[name="root_nonMilitaryJobs_0_name"]')
```

Comment:
Oh, actually I didn't write that at all, that's why. I just copied from the 1990e. I'll change it when I move it over to edu-helpers.